app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: "Teach pupils with special educational needs and disabilities"
```

Comment:
Is there any merit in including `(SEND)` in the title now that it's been removed from the h1?